packages/web/src/pages/pay-and-earn-page/components/SalesTab.tsx

```diff
@@ -187,46 +186,48 @@ export const useSales = () => {
 
       const rows = await Promise.all(
         sales.map(async (sale) => {
+          let decryptedEmail = ''
           try {
+            const decryptionId = sale.isInitial
+              ? Number.parseInt(import.meta.env.VITE_EMAIL_ENCRYPTION_UUID)
```

Comment:
can you add this to the typescript env instead
https://github.com/AudiusProject/audius-protocol/blob/3c7c4449cfb83c395a060d305fe38e948017e2d2/packages/web/src/services/env/env.prod.ts#L1

It makes for much nicer typesystem. the .env.stage and .env.prod files are used to just configure port/ts env var file

Reply:
it also allows for react native to get code push changes to env, which don't work with these env files (grant your change is web-only)